src/routes/comments.py

```
@@ -0,0 +1,96 @@
+from schemas.comments import CommentCreate, CommentOut, NotificationOut
+from security.http import get_current_user
+# adding comment to push
+
+
+router = APIRouter()
+
+
+@router.post("/", response_model=CommentOut)
+async def create_comment(
+    comment: CommentCreate,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)